dask_cuda/initialize.py

```diff
@@ -103,10 +103,15 @@ def _create_cuda_context(protocol="ucx"):
     if protocol not in ["ucx", "ucxx", "ucx-old"]:
         return
 
+    # protocol is in {"ucx", "ucxx", "ucx-old"} here
+
     try:
         ucx_implementation = _get_active_ucx_implementation_name(protocol)
     except ValueError:
         # Not a UCX protocol, just raise CUDA context warnings if needed.
+        # Claim: this is unreachable.
```

Comment:
Probably not in this PR, but I think this code is unreachable.

Reply:
This is a good catch. I think we could fix it in this PR if you don't mind, I would just remove the check in lines 103-104.